apps/mobile/src/hooks/useLogout.ts

```diff
@@ -30,7 +31,12 @@ export function useLogout(): [() => void, boolean] {
     try {
       magic.user.logout();
       provider?.disconnect();
-      const response = await mutate({ variables: {} });
+      const expoPushToken = await Notifications.getExpoPushTokenAsync();
+      const response = await mutate({
```

Comment:
does this work / not error if the user doesn't have push notifications enabled too?